app/design/adminhtml/default/default/template/sales/order/view/info.phtml

```diff
@@ -19,21 +19,16 @@
 <?php
 $orderAdminDate = $this->formatDate($_order->getCreatedAtDate(), 'medium', true);
 $orderStoreDate = $this->formatDate($_order->getCreatedAtStoreDate(), 'medium', true);
+$emailSentInfo  = $_order->getEmailSent() ? Mage::helper('sales')->__('Sent') : Mage::helper('sales')->__('Not sent');
```

Comment:
Following the rules of Backend labels, shouldn't it be Not **S**ent, with a capital S?

Reply:
Again. Thanks for suggestion, but please use https://egghead.io/lessons/github-add-suggestions-in-a-github-pr-review

Reply:
Are there any rules? Why should it be a capital S?

Reply:
@addison74 thanks....

Can you imagine that i have thougt about how to name it?

 - an "order confirmation" isnt excactly same as an "order confirmation email"
 - i took the wording we already (from box title)
 - even as non-native english-speaker capital "S" feels wrong

You know i am open for any suggestion, but please dont waste my time with that. If you want to have 99% instead of 80% please help. I do what i can ... and its still not enough?